luaclang.c

```diff
@@ -148,6 +148,147 @@ static int cursor_getkind(lua_State *L)
         return 1;
 }
 
+/*
+        Format - cur:getType()
+        Parameter - cur - Cursor whose type is to be obtained    
+        More info - https://clang.llvm.org/doxygen/group__CINDEX__TYPES.html#gaae5702661bb1f2f93038051737de20f4
+        Returns the type of the cursor
+*/
+static int cursor_gettype(lua_State *L)
+{
+        CXCursor *cur;
+        to_object(L, cur, CURSOR_METATABLE, 1);
+        luaL_argcheck(L, !clang_Cursor_isNull(*cur), 1, "Invalid cursor");
```

Comment:
In which case this happens?

Reply:
Right, we wouldn't be needing this. This is an unnecessary check, as Lua doesn't allow to index a nil value.